### ISSUE TITLE
Transformations outside the scope of the local projection coordinate system

### ISSUE THREAD

**Issue by syzdev** [proj4js/proj4js#446]
If the input coordinates are outside the valid range of the local projection coordinate system, an incorrect result will generally be returned. Is there any way to get a result as correct as possible?

**Comment by ahocevar** [comment]
No, because proj4js does not know the validity extent of the projections it transforms. You'd have to do that separately, with a check against data you'd have to extract from the EPSG database.
